pkg/wifi/iwl.go

```diff
@@ -155,14 +155,16 @@ func (w *IWLWorker) Connect(a ...string) error {
 	// it's been almost instantaneous. But, further, it needs to keep running.
 	go func() {
 		cmd := exec.CommandContext(ctx, "wpa_supplicant", "-i"+w.Interface, "-c/tmp/wifi.conf")
-		cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr //For an easier time debugging
+		var outBuffer, errBuffer bytes.Buffer
+		cmd.Stdout, cmd.Stderr = &outBuffer, &errBuffer //For an easier time debugging
 		cmd.Run()
 	}()
 
 	// dhclient might never return on incorrect passwords or identity
 	go func() {
 		cmd := exec.CommandContext(ctx, "dhclient", "-ipv4=true", "-ipv6=false", "-v", w.Interface)
-		cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr //For an easier time debugging
+		var outBuffer, errBuffer bytes.Buffer
```

Comment:
same here. Pass in an io.Writer (or two, if you want to keep cmd.Stdout and cmd.Stderr separate) -- up to you